src/PermissionResourceTrait.php

```diff
@@ -16,6 +16,17 @@
 
 trait PermissionResourceTrait
 {
+
+	/**
+	 * Override the applyFilters method,title field translation
+	 */
+	public function title() {
+
+		return array_has(__('laravel-nova-permission::permissions.display_names'),$this->name)
```

Comment:
this throws an error `array_key_exists() expects parameter 2 to be array, string given` if the `display_names` key isn't defined.